examples/exotica_examples/scripts/example_sphere_collision.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+
+import pyexotica as exo
+from pyexotica.publish_trajectory import *
+
+# Custom pose and trajectory publishing to force update the task map debug marker publishing
+def publishPose(q, problem, t=0):
```

Comment:
I think it may make sense to make this the default behaviour for the regular publishing?

Reply:
It requires integer indexing (unlike real valued time value) for the time indexed problem update. This is incompatible with publishing the stream of poses from IK.
We can declare a separate `publishTimeIndexedPose` and use that instead though.